src/transformers/modeling_utils.py

```diff
@@ -3403,6 +3458,37 @@ def _load_pretrained_model_low_mem(model, loaded_state_dict_keys, resolved_archi
         error_msgs = _load_state_dict_into_meta_model(model, state_dict, loaded_state_dict_keys, start_prefix)
         return error_msgs
 
+    @classmethod
+    # Copied from transformers.models.auto.auto_factory._BaseAutoModelClass._check_and_return_if_adapter_model
+    def _check_and_return_if_adapter_model(
```

Comment:
It's more a `_find_adapter_config_file` since it returns this if it exists and None if it doesnt.